test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -1119,11 +1120,30 @@ namespace iroha {
         hash3 = txs2.at(0).hash();
       }
 
+      void commitAdditionalBlocks(const size_t amount) {
+        shared_model::crypto::Hash prev_block_hash = second_block_hash;
+        size_t starting_height = 3;
+        for (size_t i = 0; i < amount; ++i) {
+          std::vector<shared_model::proto::Transaction> txs;
+          std::string role_name = "test_role_" + std::to_string(i);
+          txs.push_back(TestTransactionBuilder()
+                            .creatorAccountId(account_id)
+                            .createRole(role_name, {})
+                            .build());
+          auto block = createBlock(txs, starting_height + i, prev_block_hash);
+          prev_block_hash = block->hash();
+          apply(storage, block);
+          hashes.push_back(txs.at(0).hash());
+        }
+      }
+
       const std::string asset_id = "coin#domain";
       shared_model::crypto::PublicKey fake_pubkey{zero_string};
       shared_model::crypto::Hash hash1;
       shared_model::crypto::Hash hash2;
       shared_model::crypto::Hash hash3;
+      shared_model::crypto::Hash second_block_hash;
+      std::vector<shared_model::crypto::Hash> hashes;
```

Comment:
this variable looks weird together with the other ones above (they are all about some hashes!). would it be prettier to return the tx hashes in "additional blocks" form the function that creates these blocks?